tests/unit/store/playback.spec.js

```diff
@@ -602,32 +623,32 @@ describe("store/playback", () => {
       mocked_axios.onGet(all_mantarray_commands_regexp).reply(200);
 
       // confirm pre-condition
-      expect(store.state.playback.playback_state).not.toEqual(
+      expect(store.state.playback.playback_state).not.toStrictEqual(
         playback_module.ENUMS.PLAYBACK_STATES.CALIBRATING
       );
 
       await store.dispatch("playback/start_calibration");
 
-      let request_to_start_calibration = mocked_axios.history.get[0];
+      const request_to_start_calibration = mocked_axios.history.get[0];
 
       expect(request_to_start_calibration.url).toMatch(`${base_url}/${api}`);
 
-      expect(store.state.playback.playback_state).toEqual(
+      expect(store.state.playback.playback_state).toStrictEqual(
         playback_module.ENUMS.PLAYBACK_STATES.CALIBRATING
       );
 
       await wait_for_expect(() => {
         expect(
           store.state.flask.status_ping_interval_id
         ).toBeGreaterThanOrEqual(0);
-        expect(store.state.playback.playback_state).toEqual(
+        expect(store.state.playback.playback_state).toStrictEqual(
           playback_module.ENUMS.PLAYBACK_STATES.CALIBRATED
         );
       });
     });
 
-    test("Then playback state mutates to BUFFERING and starts status_pinging in Flask, then playback state mutates to LIVE_VIEW_ACTIVE", async () => {
-      let api = "start_managed_acquisition";
+    test("When playback state mutates to BUFFERING and starts status_pinging in Flask, Then playback state mutates to LIVE_VIEW_ACTIVE", async () => {
```

Comment:
Mocking /system_status to return a specific value should be included in the Given section of the test title. The trigger/When/Act for this test is dispatching start_live_view.  After that is dispatched, you are asserting that the state first mutates to buffering, and then mutates to live_view_actiev